test/integration/restart_task_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+require 'kubernetes-deploy/restart_task'
+
+class RestartTaskTest < KubernetesDeploy::IntegrationTest
+  def test_restart
+    deploy_fixtures("basic", subset: ["configmap-data.yml", "web.yml.erb"])
+
+    deployment = fetch_deployment("web")
+    refute fetch_restarted_at(deployment), "no RESTARTED_AT env on fresh deployment"
+
+    restart = KubernetesDeploy::RestartTask.new(
+      context: KubeclientHelper::MINIKUBE_CONTEXT,
+      namespace: @namespace,
+      logger: @logger
+    )
+    restart.perform(["web"])
+
+    assert_logs_match(/Restarted "web"/)
+
+    deployment = fetch_deployment("web")
+    assert fetch_restarted_at(deployment), "RESTARTED_AT is present after the restart"
+  end
```

Comment:
Is there any helper to test restarting multiple? What happens if I pass `web` twice?